---
title: Publications
---
For the current list of publications, please visit the publications page of [our main website](https://basilbiochem.github.io/basil/pubs/index.html).
**BASIL**
  1. Roberts R, Hall B, Daubner C, Goodman A, Pikaart, M, Sikora A, Craig P. Flexible Implementation of the BASIL CURE, [Biochemistry and Molecular Biology Education 47:498-505 (2019)](https://doi.org/10.1002/bmb.21287).
  1. Irby SM, Pelaez NJ, Anderson TR. Anticipated Learning Outcomes for a Biochemistry Course-based Undergraduate Research Experience Aimed at Predicting Protein Function from Structure: Implications for Assessment Design. [Biochemistry and Molecular Biology Education, 46(5), 478–492 (2018)](https://doi.org/10.1002/bmb.21173)
  1. Irby SM, Pelaez NJ, Anderson TR. How to Identify the Research Abilities That Instructors Anticipate Students Will Develop in a Biochemistry Course-Based Undergraduate Research Experience (CURE). [CBE Life Sciences Education 17:es4, 1-14 (2018)](https://www.lifescied.org/doi/full/10.1187/cbe.17-12-0250).
  1. Craig PA, Anderson T, Bernstein HJ, Daubner C, Goodman A, Irby SM, Koeppe J, Mills JL, Pikaart M, Ringer McDonald A‡, O’Handley SF, Roberts R, and Stewart R. Using Protein Function Prediction to Promote Hypothesis-Driven Thinking in Undergraduate Biochemistry Education. [The Chemist 91:1-8 (2018)](http://www.theaic.org/pub_thechemist_journals/Vol-91-No-1/Vol-91-No1-Article-1.html).
  1. Craig, PA. Lessons from my undergraduate research students. [Journal of Biological Chemistry, 293, 10447-10452 (2018)](http://www.jbc.org/content/293/27/10447.full), DOI: 10.1074/jbc.RA118.003722.
  1. Craig, PA. A survey on faculty perspectives on the transition to a biochemistry course‐based undergraduate research experience laboratory. [Biochemistry and Molecular Biology Education 45, 426-436 (2017)](https://iubmb.onlinelibrary.wiley.com/doi/full/10.1002/bmb.21060) DOI: 10.1002/bmb.21060.


**ProMOL**

  1. Osipovitch M, Lambrecht M, Baker C, Madha S, Mills JL, Craig PA, Bernstein HJ. Automated protein motif generation in the structure-based protein function prediction tool ProMOL, [Journal of Structural and Functional Genomics 16:101-11 (2015)](https://link.springer.com/article/10.1007%2Fs10969-015-9199-0) DOI: 10.1007/s10969-015-9199-0; PubMed PMID: 26573864. In process at NIHMS. NIHMS ID: NIHMS738909.
  1. McKay T, Hart K, Horn A, Kessler H, Dodge G, Bardhi K, Bardhi K, Mills JL, Bernstein, HJ, Craig PA. Annotation of Proteins of Unknown Function: Initial Enzyme Results, [Journal of Structural and Functional Genomics 16:43-54 (2015)](https://link.springer.com/article/10.1007%2Fs10969-015-9194-5) DOI: 10.1007/s10969-015-9194-5; PubMed PMID: 25630330; PubMed Central PMCID: PMC4332402.
  1. Hanson B, Westin C, Rosa M, Grier A, Osipovitch M, MacDonald ML, Dodge G, Boli, PM, Corwin CW, Kessler H, McKay T, Bernstein HJ, Craig PA. Estimation of Protein Function Using Template-Based Alignment of Enzyme Active Sites. [BMC Bioinformatics, 15:87 (2014)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-15-87), DOI:10.1186/1471-2105-15-87; PubMed PMID: 24669788; PubMed Central PMCID: PMC4229977.
  1. Grell L, Parkin C, Craig PA, Slatest L.  EZ-Viz – A Tool for Simplifying Molecular Viewing in PyMOL, [Biochemistry and Molecular Biology Education 34:402-407 (2006)](https://iubmb.onlinelibrary.wiley.com/doi/full/10.1002/bmb.2006.494034062672) DOI: 10.1002/bmb.2006.494034062672.

{% include links.md %}
